packages/engine/src/scene/components/PrimitiveGeometryComponent.ts

```diff
@@ -51,10 +51,10 @@ export const PrimitiveGeometryComponent = defineComponent({
   jsonID: 'EE_primitive_geometry',
 
   schema: S.Object({
-    geometryType: S.Enum(GeometryTypeEnum, {
+    geometryType: S.Enum(GeometryType, {
       $comment:
```

Comment:
oh, we also need to update the comments in each enum to not say it's a numerical value